### PR TITLE
Ww 3 t footbar

### PR DESCRIPTION
**Pull request by Takkunkuma**

### CODE REVIEW

File: src/Components/footer.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      <li className="py-3 px-4">
+        <a
+          href={href}
+          className="font-medium text-sm text-navWords font-['Inter'] tracking-[0.5px] desktop:font-bold desktop:text-lg"
+        >
+          {title}
+        </a>
+      </li>
+    );
+  };
```
Comment (BenjaminnHuang) [inline src/Components/footer.jsx]: this is good. Reduce the repetitive code.

File: src/Components/footer.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <footer
+      className="bg-gray-100 px-[35px] py-[62.5px] desktop:px-[120px] desktop:py-[136px]"
+      id="foot"
+    >
+      <div className="flex flex-col desktop:flex-row justify-center items-center gap-6">
+        <div>Weavers Logo</div>
+        <div className="flex flex-row w-full justify-center gap-20 desktop:justify-end desktop:gap-10">
```
Comment (BenjaminnHuang) [inline src/Components/footer.jsx]: You can just remove the "w-full" from the third div and change "justify-center" to "justify-between" in the first div.  I recommend adding border-2 to those two divs so you can see the differences.

File: src/Components/footer.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          <ul className="flex flex-col items-start justify-between">
+            {firstCol.map((item) => (
+              <FootItem key={item.href} {...item} />
+            ))}
+          </ul>
+          <ul className="flex flex-col iterms-start">
+            {secondCol.map((item) => (
+              <FootItem key={item.href} {...item} />
+            ))}
+          </ul>
```
Comment (BenjaminnHuang) [inline src/Components/footer.jsx]: Not a fan of this. The code is repetitive and hard to read. Instead, use **grid** over **flex** for this situation. Create a grid with two columns then adjust the gap between the columns based on your needs. It's a good chance to practice **grid** as well.
That way, you don't need secondCol and firstCol. You will just need one colItems data.

**Review by BenjaminnHuang — CHANGES_REQUESTED** [review]
some comments to your PR. Lmk if you need me to clarify something.

File: src/Components/footer.jsx (hunk 2 above)
Comment (Takkunkuma) [reply src/Components/footer.jsx]: Resolved!

File: src/Components/footer.jsx (hunk 3 above)
Comment (Takkunkuma) [reply src/Components/footer.jsx]: Changed and pushed!

File: src/Components/footer.jsx (hunk 1 above)
Comment (Takkunkuma) [reply src/Components/footer.jsx]: Thank you!

**Pull request merged**